ui/apps/everest/.e2e/utils/table.ts

```diff
@@ -6,11 +6,13 @@ export const findRowAndClickActions = async (
   nameOfAction?: string
 ) => {
   // cluster actions menu click
-  await page
+  const dbRow = page
     .locator('.MuiTableRow-root')
     .filter({ hasText: name })
-    .getByTestId('MoreHorizIcon')
-    .click({ timeout: 10000 });
+    .first();
```

Comment:
Which text is appearing twice? I don't like this because `.first()` is not deterministic and you don't know what is going to end up first so it is better that the test fails if there are some unexpected strings.
If we can control cluster names make them unique.
If substring is a problem consider using regex like `.filter({ hasText: name RegExp(`^${name}$`) })`.